internal/data/disbursement_instructions_test.go

```diff
@@ -202,7 +311,6 @@ func Test_DisbursementInstructionModel_ProcessAll(t *testing.T) {
 			Instructions:            emailAndSMSInstructions,
 			Disbursement:            disbursement,
 			DisbursementUpdate:      disbursementUpdate,
-			ReceiverContactType:     ReceiverContactTypeEmail,
```

Comment:
Did you really mean to delete this one? The title for this subtest refers to email but you removed the email enum.

Reply:
Yes, because we're not passing this parameter as part of `DisbursementInstructionsOpts`, it's now taken directly from the Disbursement.